### PR TITLE
Add "RELEASE_NAME" to BUILD.md

### PR DESCRIPTION
**Pull request by digisomni**

### CODE REVIEW

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1014/Vircadia-PR1014-578d45a.tar_b995d6d4.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1014/Vircadia-PR1014-578d45a_b995d6d4.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1014/Vircadia-PR1014-578d45a.tar_b995d6d4.Z

**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1014/Vircadia-PR1014-578d45a_b995d6d4.dmg

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1014/Vircadia-PR1014-578d45a_b995d6d4.exe

**build (ubuntu-18.04, android)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1014/Vircadia-Quest-Alpha-PR1014-578d45a_b995d6d4.apk
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1014/Vircadia-PR1014-578d45a_b995d6d4.apk

File: BUILD.md [hunk 1]
```diff
@@ -72,9 +72,11 @@ Where `/path/to/directory` is the path to a directory where you wish the build f
     // The identifying tag of the release.
     CMAKE_BACKTRACE_TOKEN
     
-    // The release version.
+    // The release version. e.g. 2021.3.2
```
Comment (ctrlaltdavid) [inline BUILD.md:75]:
```suggestion
    // The release version, e.g., 2021.3.2.
```

File: BUILD.md [hunk 2]
```diff
@@ -72,9 +72,11 @@ Where `/path/to/directory` is the path to a directory where you wish the build f
...
     RELEASE_NUMBER
-    // The build commit.
+    // The release name. e.g. Eos
```
Comment (ctrlaltdavid) [inline BUILD.md:77]:
```suggestion
    // The release name, e.g., Eos.
```

File: BUILD.md [hunk 3]
```diff
@@ -72,9 +72,11 @@ Where `/path/to/directory` is the path to a directory where you wish the build f
...
+    RELEASE_NAME
+    // The build commit. e.g. Use a Git hash for the most recent commit in the branch: fd6973b
```
Comment (ctrlaltdavid) [inline BUILD.md:79]:
```suggestion
    // The build commit, e.g., use a Git hash for the most recent commit in the branch - fd6973b.
```

**Review by ctrlaltdavid — CHANGES_REQUESTED** [review]

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1014/Vircadia-PR1014-2695e45_e64c9ad0.dmg

**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1014/Vircadia-PR1014-2695e45_e64c9ad0.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1014/Vircadia-PR1014-2695e45.tar_e64c9ad0.Z
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1014/Vircadia-PR1014-2695e45.tar_e64c9ad0.gz

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1014/Vircadia-PR1014-2695e45_e64c9ad0.exe

**build (ubuntu-18.04, android)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1014/Vircadia-PR1014-2695e45_e64c9ad0.apk
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1014/Vircadia-Quest-Alpha-PR1014-2695e45_e64c9ad0.apk

**Review by ctrlaltdavid — APPROVED** [review]

**Pull request merged**